inference-engine/src/vpu/graph_transformer/src/middleend/passes/propagate_dynamism_to_outputs.cpp

```diff
@@ -39,34 +39,33 @@ class PassImpl final : public Pass {
             }
             const auto parent = parentDataToShapeEdge->parent();
 
+            const auto& parentAttrs = parent->attrs();
+            VPU_THROW_UNLESS(parentAttrs.has("converted-notation") && parentAttrs.get<bool>("converted-notation"),
```

Comment:
`getOrDefault("converted-notation", false)` would be more briefly

Reply:
Nice catch, fixed!